shotover/src/runner.rs

```diff
@@ -45,6 +45,10 @@ struct ConfigOpts {
 
     #[arg(long, value_enum, default_value = "human")]
     pub log_format: LogFormat,
+
+    //hot reloading functionality
```

Comment:
If we use three `/`s the comment will become part of the documentation generated when running `--help`.
lets make use of that to document the flag.
e.g.
![image](https://github.com/user-attachments/assets/f2a5889b-c66e-43ed-9551-5b522c593752)
results in:
![image](https://github.com/user-attachments/assets/0fca7f20-9461-464c-9d14-289fb8c6cdcf)



The existing flags dont have any documentation, those flags are very old code that doesnt get much attention, one day we should add documentation for them but documenting them is out of scope of this PR.